src/main/java/org/jabref/gui/maintable/MainTableDataModel.java

```diff
@@ -26,20 +26,23 @@
 import com.tobiasdiez.easybind.EasyBind;
 
 public class MainTableDataModel {
-    private final FilteredList<BibEntryTableViewModel> entriesFiltered;
```

Comment:
I think this needs be a class variable, since otherwise the bindings / listeners are deleted by the garbage collector.